doc/classes/Object.xml

```diff
@@ -115,7 +119,7 @@
 			<argument index="4" name="flags" type="int" default="0">
 			</argument>
 			<description>
-				Connect a signal to a method at a target (member function). Binds are optional and are passed as extra arguments to the call. Flags specify optional deferred or one shot connections, see enum CONNECT_*. A signal can only be connected once to a method, and it will throw an error if already connected. If you want to avoid this, use [method is_connected] to check.
+				Connects a [code]signal[/code] to a [code]method[/code] within a [code]target[/code] Object. Pass optional [code]binds[/code] to the call. Specify optional [code]flags[/code] for deferred or one shot connections. See [code]CONNECT_*[/code] constants. A [code]signal[/code] can only be connected once to a [code]method[/code]. It will throw an error if already connected. To avoid this, check using the method [code]is_connected[/code].
```

Comment:
- Below you use "on a target", use that here instead of "within".
- Use [code]flags[/code] to set deferred or one shot connections. See [code]CONNECT_*[/code] constants.
- To avoid this, first use [method is_connected] to check for existing connections.